package.json

```diff
@@ -57,6 +57,7 @@
   },
   "dependencies": {
     "chalk": "2.4",
+    "json-schema-merge-allof": "^0.8.1",
```

Comment:
noticing just now, but don't we technically need `lodash` and `cheerio` as dependencies, instead of devDependencies ?